maps/astrafare/warfare.dm

```diff
@@ -0,0 +1,28 @@
+#if !defined(using_map_DATUM)
+	#include "warfare_areas.dm"
+	#include "warfare_shuttles.dm"
+	#include "warfare_unit_testing.dm"
+	#include "jobs/captain_verbs.dm"
+	#include "jobs/warfare_jobs.dm"
```

Comment:
If you didn't actually make any changes to the jobs themselves, then these can also just be removed and we can use the oldfare ones.

Reply:
Agreed.